vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
+Entando Customer Portal is a self-service application for customers to interact within your organization to administer your Entando platform and its users. You can manage users, projects and customers. You can track projects and service tickets for your application. It is a Kubernetes-native solution built on Entando’s Application Composition Platform.
+
+Key Features:
+
+* Customizable service ticketing and tracking system with Jira Service management
+* Role-Based Access Control (RBAC)  Entando Identity Management System
+* Integrated User, customer, project and subscription management 
+
+This tutorial covers 
+1. [Installation](#installaton)
+2. [Configuration for Administrators](#configuration)
+3. [A brief User Reference](#user-refernce)
+
+# Prerequisites
+* A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/)for more information.
+* Use the Entando CLI to verify that all dependencies are installed:
+ent check-env develop
+* Docker installed and running
+
+# Installation
+## Quick Install from Entando Hub
+
+1. From the Entando Hub, click on the Customer Portal bundle from the Catalog
+2. Click Install to see a popup page with two commands. These need to be implemented in the order shown. Copy the first command and paste into your CLI. 
+3. Copy the second command and do the same. The content bundle will now be deployed.
+4. Go to the Entando App Builder and navigate to the Repository.  Here you will see the two bundles for the Portal. Install both bundles in the version you desire. You will be able to update the version at any time.
+5. To navigate to your Customer Portal: <br>
+from the left menu →  `Page` → `Mangement` <br>
+find `Customer Portal` under `Page Tree` list<br>
+from the `Actions` pull-down menu →  `View Published Page`
+
+## Manual Install 
+Top install the Entando Customer Portal manually to your specific environment, go to the [Entando Github Repository](https://github.com/entando-samples/customer-portal/tree/main/application) and refer to the README.MD there for more details.
+
+# Configuration
+## Administrators
+In order to configure the Customer Portal and its users, the administrator will need Jira Service Manager and Entando Identity Management credentials. As the administrator, you can create and configure users and groups, assign roles, projects and track service tickets.  You can also customize the ticketing system. 
+
+### JIRA Service Manager 
+
+The administrator begins here to create users and projects, define the organization, and configure the service ticket system.
+
+Go to Customers to add organizations and projects. Once added, click on the name of the organization to get the ID from the URL which is needed later, e.g example.com/jira/servicedesk/projects/ECS/organization/3, where the organization ID is “3”.
+
+For more information or help with the Jira Service Management System, contact ??????
```

Comment:
lol maybe don't offer a lifeline!

Reply:
Agreed, or maybe have a Resources section with a link to JIRA Service Management - https://www.atlassian.com/software/jira/service-management. Also to the source code, at a minimum.

Reply:
Ok will add links under Resources